api/saturating_add_pos.md

```diff
@@ -0,0 +1,36 @@
+---
+api_name: saturating_add_pos()
+excerpt: Adds two numbers, saturating at 0 for the minimum bound
+license: community
+toolkit: true
+topic: hyperfunctions
+tags: [hyperfunctions, saturating math]
+api_category: hyperfunction
+experimental-toolkit: true
+hyperfunction_toolkit: true
+hyperfunction_family: 'saturating math'
+hyperfunction_subfamily: saturating math
+hyperfunction_type: one-step
+---
+
+# saturating_add_pos()  <tag type="toolkit">Toolkit</tag><tag type="toolkit-experimental" content="Experimental" />
+The `saturating_add_pos` function adds two numbers, saturating at 0 for the minimum bound.
```

Comment:
This also saturates the upper bound to INT_MAX right?  Should probably mention that if so.